### PR TITLE
Filter success results from data

### PR DESCRIPTION
**Pull request by r0x0d**
If we find messages or actions with success results, we will filter them out and remove from the returning data. We only want status that are higher than that.

### CODE REVIEW

File: tests/preconversion_assessment/test_format.py [hunk 1]
```diff
@@ -251,9 +236,53 @@ def test_apply_message_transform(data, action_id, expected):
...
 def test_transform_raw_data(data, expected):
     result = transform_raw_data(data)
-    print(pprint(result))
+    assert result == expected
+
+
+@pytest.mark.parametrize(
+    ("data", "level", "expected"),
+    (
+        ({"level": "SUCCESS"}, "SUCCESS", None),
```
Comment (andywaltlova) [inline tests/preconversion_assessment/test_format.py:282]: What is expected? `{}` or `None`? test is failing

**Review by andywaltlova — APPROVED** [review]
Just one question otherwise it makes sense

File: tests/preconversion_assessment/test_format.py (hunk 1 above)
Comment (r0x0d) [reply tests/preconversion_assessment/test_format.py:282]: Empty dictionary 😓. Changed the return value for the function and forgot about the unit test.

Comment (r0x0d) [reply tests/preconversion_assessment/test_format.py]: done.

**Pull request merged**